app/javascript/article-form/components/EditorBody.jsx

```diff
@@ -70,6 +70,13 @@ export const EditorBody = ({
     }
   });
 
+  // In some platforms we've seen `navigator.clipboard` return `undefined`
+  // despite `window.isSecureContext` being true. A refresh fixes the problem.
+  // It's important to only do this when isSecureContext (not in dev/test env)
```

Comment:
Does it make sense to add context from this PR to the comment?

```suggestion
  // It's important to only do this when isSecureContext (not in dev/test env).
  // This code runs before the Editor renders, so the likelihood of a user
  // losing draft edits is low.
```